src/styles/blocks.scss/header.scss

```diff
@@ -0,0 +1,159 @@
+.header {
+  background-color: #3f5252;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    height: 100vh;
+    padding: 24px 0;
+    box-sizing: border-box;
+  }
+
+  &__info-container {
+    padding-bottom: 280px;
+
+    @include onDesktop {
+      padding-bottom: 0;
+    }
+  }
+
+  &__title {
+    display: block;
+    align-items: center;
+    position: relative;
+    font-family: "Playfair Display", sans-serif;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 100%;
+    color: $textColor;
+    max-width: 700px;
+    padding: 10px;
+
+    @include onTablet {
+      font-weight: 700;
+      font-size: 72px;
+
+      @include onDesktop {
+        font-weight: 700;
+        font-size: 96px;
+      }
+    }
+
+    &::before {
+      content: "10 серпня - 10 листопада";
+      position: absolute;
+      top: -30px;
+      font-family: "Raleway", serif;
+      font-weight: 400;
+      font-size: 14px;
+      line-height: 20px;
+
+    }
+
+    &-subtitle {
+      font-family: "Raleway", serif;
+      font-weight: 400;
+      font-size: 16px;
+      line-height: 150%;
+      color: #fff;
+      padding-bottom: 20px;
+    }
```

Comment:
Why. do you use such a nesting here `&-`?

You could nest modifiers, or elements, but there you have non of them.

Watch more about naming convention in BEM [in this video](https://mate.academy/learn/css-advanced/bem#/video/920)
Watch closely when you use '-' in naming.


Please check this moments all over the project.